extensions/wikia/CommunityPage/styles/components/_rail.scss

```diff
@@ -10,6 +10,7 @@ $community-page-rail-background-color: mix($color-page, #000, 96%);
 }
 $community-page-rail-module-width: 210px;
 $community-page-rail-module-spacing: 35px;
+$community-page-rail-module-spacing-left: 24px;
 $community-page-rail-spacing-small: 14px;
 $community-page-rail-spacing: 16px;
 $community-page-rail-spacing-large: 18px;
```

Comment:
Only this single one is actually reusable. That's why I decide to leave it as is.